packages/fxa-settings/src/pages/ResetPasswordRedesign/ResetPasswordStart/interfaces.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { QueryParams } from '../../..';
+import {
+  BaseIntegration,
+  IntegrationType,
+  OAuthIntegration,
+} from '../../../models';
+import { MozServices } from '../../../lib/types';
+
+export interface ResetPasswordOAuthIntegration {
+  type: IntegrationType.OAuth;
+  getRedirectUri: () => ReturnType<OAuthIntegration['getRedirectUri']>;
```

Comment:
We probably want to use `Pick` instead here, see the Signin interfaces like `type SigninOAuthIntegration`. I started doing it the way you have it and then Dan used `Pick` and I was like why didn't I do that originally, so I blame myself for this pattern being around😂

Reply:
Oh yeah, that's much nicer.